src/types.ts

```diff
@@ -143,6 +143,108 @@ export interface UpdateOptions {
   // session?: ClientSession
 }
 
+/**
+ * interface for WriteConcern documents used by MongoDB
+ *
+ * @see https://docs.mongodb.com/manual/reference/write-concern/
+ */
+export interface WriteConcern {
+  /**
+     * The number of instances the write operation needs to be propagated to
+     * before proceeding.
+     *
+     * The string based values are:
+     *
+     * - majority: The calculated majority of nodes in a cluster has accepted the
+     *    the write
+     * - custom write name: Writes have been acknowledged by nodes tagged with the
+     *    custom write concern.
+     */
+  w: number | "majority" | string;
+  /**
+     * If true, the server only returns after the operation has been commited to
+     * disk
+     */
+  j: boolean;
+  /**
+     * An optional timeout value after which to stop the write operation
+     */
+  wtimeout?: number;
+}
+
+/**
+ * Options for controlling the collation of strings in a query
+ *
+ * @see https://docs.mongodb.com/manual/reference/collation/
+ */
+export interface CollationOptions {
+  locale: string;
+  caseLevel?: boolean;
+  caseFirst?: string;
+  strength?: number;
+  numericOrdering?: boolean;
+  alternate?: string;
+  maxVariable?: string;
+  backwards?: boolean;
+}
+
+/**
+ * Options for the findAndModify operation
+ *
+ * @see https://docs.mongodb.com/manual/reference/method/db.collection.findAndModify/
+ */
+export interface FindAndModifyOptions<T> {
+  /**
+     * Control the order in which documents are found.
```

Comment:
align those comment

Reply:
Fixed. Those must've slipped under the radar.